src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+spring:
+  profiles:
+    active: default
+  datasource:
+    url: ${SPRING_DATASOURCE_URL:jdbc:postgresql://localhost:5432/beacons}
+    username: ${SPRING_DATASOURCE_USER:beacons}
+    password: ${SPRING_DATASOURCE_PASSWORD:password}
```

Comment:
See: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-connect-to-production-database